packages/fixie/package.json

```diff
@@ -75,5 +74,10 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "files": ["dist"],
+  "exports": {
+    ".": "./dist/main.js",
+    "./web": "./dist/web.js"
```

Comment:
@NickHeiner you expressed some FUD around this approach -- is there a particular scenario that won't work/I should test?

Reply:
I would try this:

1. Publish a prerelease version (`1.8.0-0`)
2. Try to use this package from `fixie/src/frontend`
3. Try to use this package from `fixie/src/generic-sidekick-frontend`
4. Try to use this package from https://github.com/fixie-ai/redwood-fixie-sample

If it works in all those places, ✨🎵 tHiNgS sHoUlD bE fInE 🎵 ✨